client/src/App.js

```diff
@@ -11,7 +14,12 @@ function App() {
   return (
     <>
       <GlobalStyles />
-      <p>Hola Papa!</p>
+      <h2>Headline</h2>
```

Comment:
You cannot have a h2 tag unless there's a h1 tag beforehand. is it to big? style id ;-)

```suggestion
      <h1>Headline</h1>
```